Solutions/PaloAltoPrismaCloud/Hunting Queries/PaloAltoPrismaCloudTopResources.yaml

```diff
@@ -0,0 +1,25 @@
+id: 08ab5107-5c4e-4baf-b0f4-bf75c044f8b1
+name: Palo Alto Prisma Cloud - Top recources with alerts
+description: |
+  'Query searches for resources which appeared in different alerts.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: PaloAltoPrismaCloud
+    dataTypes:
+      - PaloAltoPrismaCloud
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1133
+query: |
+  PaloAltoPrismaCloud
+  | where TimeGenerated > ago(24h)
+  | where Reason =~ 'NEW_ALERT'
+  | summarize count() by ResourceName
+  | order by count_ desc
+  | extend AccountCustomEntity = UserName

```

Comment:
@vu-socprime ,  This query is not working , In order to fix this please move" | extend AccountCustomEntity = UserName" before "| Summarize ....".[PFB updated query]

  PaloAltoPrismaCloud
  | where TimeGenerated > ago(24h)
  | where Reason =~ 'NEW_ALERT'
  **| extend AccountCustomEntity = UserName**
  | summarize count() by ResourceName
  | order by count_ desc

Reply:
updated in new PR - https://github.com/Azure/Azure-Sentinel/pull/2132

Reply:
@vu-socprime , We have already merged this fix, can you please check and confirm . [PR 2126](https://github.com/Azure/Azure-Sentinel/pull/2126).

Reply:
We need to change UserName to ResourceName. I have made changes in https://github.com/Azure/Azure-Sentinel/pull/2132

Reply:
ok Thanks !